rfcs/text/0000-data-source-categorization-fields.md

```diff
@@ -0,0 +1,75 @@
+# 0000: Data Source Categorization Fields
```

Comment:
In Ingest Management we had many iterations on naming on data source has also some history in it. I'm wondering what exactly we categorise here. Is it the data itself which is in data_streams? Do we category the data_streams? Do we categorize the source from where the data is coming from?

Reply:
Hi @ruflin - the intent here is to categorize the source from where the data is coming from.